src/lib.rs

```diff
@@ -713,9 +713,12 @@ impl Parser {
         ignoretz: bool,
         tzinfos: &HashMap<String, i32>,
     ) -> ParseResult<(NaiveDateTime, Option<FixedOffset>, Option<Vec<String>>)> {
-        let default_date = default.unwrap_or(&Local::now().naive_local()).date();
-
-        let default_ts = NaiveDateTime::new(default_date, NaiveTime::from_hms_opt(0, 0, 0).unwrap());
+        // If default is none, 1970-01-01 00:00:00 as default value is better.
```

Comment:
`dateutil` parser specifically uses the current time as a reference, I don't want to change that. Please keep the default as-is.